pythontabcmd2/commands/extracts/extracts_command.py

```diff
@@ -0,0 +1,49 @@
+from ..commands import Commands
+
+
+class ExtractsCommand(Commands):
+    """
+    Base class for extracts group of commands
+    """
+    def __init__(self, args):
+        super().__init__(args)
+
+    @staticmethod
```

Comment:
these methods seem useful for multiple commands, like delete and publish. Do we have a general place they can go?

Reply:
Yes, I moved it to the commands.py file that has the `class Commands`. All commands inherit this class so it seems like a good place.